test/test.ts

```diff
@@ -88,7 +90,7 @@ for (const tokenizerType of tokenizerTests) {
         const rst = await getTokenizerWithData('\x01\x02\x03\x04\x05\x06', tokenizerType);
         const len = await rst.readBuffer(buffer, {position: 1});
         assert.strictEqual(len, 5, 'return value');
-        assert.strictEqual(uint8ArrayToString(buffer, 'latin1'), '\x02\x03\x04\x05\x06');
```

Comment:
we added support for various encodings including `latin1` in `uint8array-extras` v1.2.0

Reply:
Nice, okay, my IDE was telling the second attribute was invalid, maybe old typings somewhere.

Can you you please strip-of my commit and rebase,?
 hope the project is now building on in the PR, after c39057d08bcf179b1b55cdbfd3001def1416628a

Reply:
Done, just need your approval to run the workflow.